embed/config.go

```diff
@@ -136,6 +136,13 @@ type Config struct {
 	//	}
 	//	embed.StartEtcd(cfg)
 	ServiceRegister func(*grpc.Server) `json:"-"`
+
+	// auth
+
+	TokenType      string
```

Comment:
It's already "simple" for etcdmain. Probably use that? Then the `fs.StringVar` declaration can load the default via  `cfg.TokenType` similar to the code around it.